litmus-portal/frontend/src/components/Header/NotificationDropdown.tsx

```diff
@@ -111,8 +113,8 @@ function NotificationsPopperButton(props: NotifierProps) {
   );
 }
 
-NotificationsPopperButton.propTypes = {
+NotificationsDropdown.propTypes = {
```

Comment:
Do you need to write it like this? You already have a NotificationsDropwdownProps Interface at the top, and message param is already present there as required. You can just modify the type to Array of Object right there

Reply:
This is something you did in JavaScript actually, with TS you have more control over your types so although this might work this isn't ideally the right practice I think.